frontend/src/clients/ReportClient.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpClient } from '@src/clients/Httpclient'
+import { AxiosError, HttpStatusCode } from 'axios'
+import { InternalServerException } from '@src/exceptions/InternalServerException'
+import { reportResponseProps } from '@src/types/reportResponse'
+
+export class ReportClient extends HttpClient {
+  reportResponse: reportResponseProps = {
+    velocity: {
+      velocityForSP: '',
+      velocityForCards: '',
+    },
+  }
+
+  generateReporter = async () => {
+    try {
+      await this.axiosInstance.post(`/report`).then((res) => {
+        this.reportResponse = res.data
+      })
+    } catch (e) {
+      const code = (e as AxiosError).response?.status
```

Comment:
How about other code? Will we ignore them?

Reply:
The backend needs to return code to the front-end, and the frontend needs to do some processing based on this code

Reply:
It looks like we only handle `code === HttpStatusCode.InternalServerError`